.gitignore

```diff
@@ -45,7 +45,7 @@ package-lock.json
 # Tools
 ## Kani
 *.out
-
+kani-list.*
```

Comment:
I'd rather have the script remove the file once it is done. I don't really see why such a file should continue to exist.